client/my-sites/exporter/post-type-options.jsx

```diff
@@ -35,7 +39,8 @@ const mapStateToProps = ( state, ownProps ) => {
 
 const mapDispatchToProps = ( dispatch, ownProps ) => {
 	return {
-		onSelect: () => dispatch( setPostType( ownProps.postType ) )
+		onSelect: () => dispatch( setPostType( ownProps.postType ) ),
+		setPostTypeFilters: ( ...args ) => dispatch( setPostTypeFilters( ...args ) ),
 	};
 };
```

Comment:
Fun fact!

The fat-arrow syntax allows implicitly returning values if it doesn't point to a code block. That means for simple cases, we can write things like `x => x * 2`, but it works for more complicated object too - "objects" are the only special ones because they share the curly-brackets with functions: they need parentheses around them.

```js
const mapDispatchToProps = ( dispatch, ownProps ) => ( {
	onSelect: …,
	setPostTypeFilters: …
} );
```